src/quo/components/list/item.cljs

```diff
@@ -223,7 +225,7 @@
      [component
       (merge {:type                :list-item
               :disabled            disabled
-              :bg-color            active-background
+              :bg-color            (when active-background-enabled active-background)
```

Comment:
Carl has made updates to the design system. We now have cards ui. I tried to re-purpose the list-item to act as a card. The only problem was that `card`s have rounded corners but `list-item`s don't.

This change removes the on-press background color. If this doesn't exist, this is what the on-press interaction will look like (observe the odd borders, because it doesn't fill the container).

https://user-images.githubusercontent.com/1925158/148245458-c0878c7f-0123-4035-9148-fb63b36c789d.mov